static/css/input.css

```diff
@@ -25,9 +39,6 @@ html.dark .svg-display svg:not(.main-svg):not(.icon) {
 div.dark\:prose-invert th {
   color: black !important;
 }
-div.dark\:prose-invert tr:hover {
-  color: black !important;
-}
```

Comment:
Please also remove the black foreground color for `<th>` in dark mode. It's not on `:hover`, but still the same effect as #109.